orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/adapters/TaskTasklet.groovy

```diff
@@ -91,13 +91,6 @@ class TaskTasklet implements Tasklet {
 
         logResult(result, stage, chunkContext)
 
-        // we should reload the execution now, in case it has been affected
-        // by a parallel process
-        long scheduledTime = stage.scheduledTime
```

Comment:
I couldn't think of a legitimate reason why this was necessary, there's no such thing as a "parallel" process that would be attempting to save the same stage.

This was possibly an issue awhile back when all stages were saved at the same time.

With this in place, it prevents the RetryableTaskTasklet from doing a `stage.self.status = PAUSED` as the stage will be reloaded.